sklearn/metrics/scorer.py

```diff
@@ -294,7 +294,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
     >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
     >>> ftwo_scorer
     make_scorer(fbeta_score, beta=2)
-    >>> from sklearn.grid_search import GridSearchCV
+    >>> from sklearn.model_selection.search import GridSearchCV
```

Comment:
Haha :D I don't mind, but there are others subscribed to this thread too. I just think pull request line comments are not the best way to leave notes to self.